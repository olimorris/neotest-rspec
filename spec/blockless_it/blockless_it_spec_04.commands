:13
:lua require("neotest").run.run()
