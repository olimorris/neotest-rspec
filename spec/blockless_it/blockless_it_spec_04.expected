./SPEC/BLOCKLESS_IT/BLOCKLESS_IT_SPEC.RB@@PASSED@@is expected to eq 17
